### ISSUE TITLE
Missing view about all Projects in the Backstage

### ISSUE THREAD

**Issue by openp2pdesign** [fablabbcn/fablabs.io#386]
We need [a view](https://github.com/fablabbcn/fablabs.io/tree/master/app/views/backstage) for _superadmins_ in the _Backstage_, in order to show all the `Projects` and enable them to edit them.

**Comment by neshoba78** [comment]
Does this include the ability to hide projects that may be spam submissions?

**Comment by openp2pdesign** [comment]
Yes, exactly... and for editing and deleting them as well, this cannot be done at the moment!

**Comment by vierarb** [comment]
Now we can access projects from the backstage side:

- The index screen, where all of them are listed.
- The show screen, to show the details of a given project.
- The edit screen, to update the details.

Although all the backend work has been done, we are still missing some frontend make up here in order to show the right fields / headers / styles. I've set a `debug(@project)` in each view to help frontenders know which fields the object has.

@openp2pdesign should I send a Pull Request or wait until the full task is completed? Also, it would be good to assign this task to the developer who is going to be in charge of it.

Also, I thought the `Hide Project` feature was out of this scope, so I'll create a new issue to cover this and will take care of it.

**Comment by viktorsmari** [comment]
@vierarb @openp2pdesign - This was never turned in to a PR. I was cleaning the branches and noticed this one had +4 commits
